avocado/plugins/run.py

```diff
@@ -77,7 +77,10 @@ def configure(self, parser):
                                   'You can also use suffixes, like: '
                                   ' s (seconds), m (minutes), h (hours). '))
 
-        parser.add_argument('--failfast', action='store_true',
+        parser.add_argument('--failfast',
+                            nargs='?',
+                            const=True,
+                            type=self._failfast,
                             help='Interrupt job on first failed test.')
```

Comment:
I see the value in `--without-failfast`, since a test after `--failfast` would not break the user expectation.